eq-author/src/App/QuestionnaireDesignPage/index.js

```diff
@@ -102,6 +106,7 @@ export const QuestionnaireDesignPage = () => {
                           questionnaire?.themeSettings?.validationErrorInfo
                             ?.totalCount > 0
                         }
+                        themeErrorCount={themeErrorCount}
```

Comment:
```suggestion
                        themeErrorCount={formTypeErrors}
```

If you do the above.